src/features/defi/providers/foxy/components/FoxyManager/Overview/Claim/ClaimStatus.tsx

```diff
@@ -121,11 +123,15 @@ export const ClaimStatus: React.FC<ClaimStatusProps> = ({ accountId }) => {
       try {
         const transactionReceipt = await poll({
           fn: () => foxyApi.getTxReceipt({ txid }),
-          validate: (result: TransactionReceipt) => !isNil(result),
+          validate: (result: ethers.providers.TransactionReceipt) => !isNil(result),
```

Comment:
if we have the transaction receipt, it should include the gas price already as an improvement here

Reply:
Absolutely, very nice catch! We already have access to `gasUsed` so that's effectively duplicate data querying our node - 6f6cb6777bd937c9a498160b7a1ad8d06af17e4a